extension/content.js

```diff
@@ -128,7 +128,7 @@ function addDeleteForkLink() {
 
 	if ($postMergeContainer.length > 0) {
 		const $postMergeDescription = $postMergeContainer.find('.merge-branch-description');
-		const forkPath = $postMergeContainer.attr('data-channel').split(':')[0];
+		const forkPath = `${getUsername()}/${repoName}`;
```

Comment:
Pushed a fix that now uses the original branch URL provided by GH in the DOM. I think this solves the possible scenario of accounts `A`, `B`, and organization `O` since it doesn't rely on you being logged in to any particular account.